server/db/models/login.js

```diff
@@ -27,6 +27,11 @@ module.exports = function (sequelize, DataTypes) {
 				allowNull: true,
 				defaultValue: ''
 			},
+			attempt: {
+				type: DataTypes.NUMBER,
```

Comment:
You defined it as integer in the migration but its NUMBER here. this should should to integer @fetok12